src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -43,13 +43,13 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
 
   lazy private[overwatch] val clusterStateFactModule = Module(3005, "Gold_ClusterStateFact", this, Array(1005, 2014), 3.0)
   lazy private val appendClusterStateFactProccess = ETLDefinition(
-    BronzeTargets.clusterEventsTarget.asIncrementalDF(clusterStateFactModule, "timestamp"),
+    BronzeTargets.clusterEventsTarget.asIncrementalDF(clusterStateFactModule, 7, "timestamp"),
```

Comment:
until version 0.5.1 running clusters with unchanging states will result in orphaned events and ultimately lead to missing costs. This addresses the 99.99%, the rest will be handled in 0.5.1 release.